src/concepts/backends/alternate-screen.md

```diff
@@ -11,6 +11,96 @@ other terminal content.
 This creates a seamless transition between the application and the regular terminal session, as the
 content displayed before launching the application will reappear after the application exits.
 
+Take this "hello world" program below. If we include the
+`std::io::stderr().execute(EnterAlternateScreen)?` (and the corresponding `LeaveAlternateScreen`),
+you can see how the program behaves differently.
+
+```rust
+use crossterm::{
+  terminal::{EnterAlternateScreen, LeaveAlternateScreen},
+  ExecutableCommand,
+};
+use ratatui::{prelude::*, widgets::*};
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+  let should_use_alternate_screen = if let Some(should_use_alternate_screen) = std::env::args().nth(1) {
+    should_use_alternate_screen.parse::<bool>()?
+  } else {
+    return Err(format!("Unable to get input argument").into());
+  };
```

Comment:
Simplify to a one liner with `.and_then()` / `unwrap_or_default()` / something similar or better yet omit?

Reply:
I removed it from the code. The code that's run for the gif is not exactly what is shown here but that's fine as long as a user is able to copy paste and run this in their project.